sky/cli.py

```diff
@@ -3629,15 +3634,27 @@ def spot():
     pass
 
 
-_add_command_alias(job,
-                   job_launch,
-                   hidden=False,
+_add_command_alias(jobs,
+                   jobs_launch,
                    new_group=spot,
                    override_command_argument={'use_spot': True})
-_add_command_alias(job, job_queue, hidden=False, new_group=spot)
-_add_command_alias(job, job_logs, hidden=False, new_group=spot)
-_add_command_alias(job, job_cancel, hidden=False, new_group=spot)
-_add_command_alias(job, job_dashboard, hidden=False, new_group=spot)
+_add_command_alias(jobs, jobs_queue, new_group=spot)
+_add_command_alias(jobs, jobs_logs, new_group=spot)
+_add_command_alias(jobs, jobs_cancel, new_group=spot)
+_add_command_alias(jobs, jobs_dashboard, new_group=spot)
+
+
+@cli.group(cls=_NaturalOrderGroup, hidden=True)
+def job():
+    """Alias for Managed Jobs CLI."""
+    pass
+
+
+_add_command_alias(jobs, jobs_launch, new_group=job, with_warning=False)
+_add_command_alias(jobs, jobs_queue, new_group=job, with_warning=False)
+_add_command_alias(jobs, jobs_logs, new_group=job, with_warning=False)
+_add_command_alias(jobs, jobs_cancel, new_group=job, with_warning=False)
+_add_command_alias(jobs, jobs_dashboard, new_group=job, with_warning=False)
```

Comment:
I'd prefer to not add the alias group for now. To keep all usage / examples clean, and to avoid the potential confusion?

Reply:
I don't have a strong feeling for this. Just removed it for now, though I am a bit used to `sky job launch` (maybe just I am too familiar).